### ISSUE TITLE
drive list is stuck on ---more---

### ISSUE THREAD

**Issue by jean-christophe-manciot** [odeke-em/drive#794]
When the pagination feature is triggered, there is no obvious or Linux-like way to continue the list:
```
# drive list
/Missions/Certifications/Cisco/CCDP/ARCH/Jean-Christophe's Guides/Summary
/Missions/Certifications/Cisco/CCDP/ARCH/Jean-Christophe's Guides/ASA & ASAv Major Features
...
---More--- 
---More---
---More---
---More--- 
---More---^C
```
I tried:
- space: a space appears
- CR: another --more--- appears
- any other character: that character appears on screen

Furthermore, in order to be consistent with Linux behavior (at least on the distributions that I know):
- a space should output another page 
- a CR should output just one more line

**Comment by odeke-em** [comment]
Hello @jean-christophe-manciot, what version of drive are you using?

**Comment by jean-christophe-manciot** [comment]
```
drive version
drive version: 0.3.8.1
Commit Hash: 'aa79e324a22a2073b68c61783f4015426b7dbe67'
Go Version: go1.7.1
OS: linux/amd64
BuildTime: 2016-10-19 06:46:40.943196228 +0200 CEST
```

**Comment by jean-christophe-manciot** [comment]
I've just rebuilt drive with latest commit (11/07) with go 1.7.3, but:
```
drive version
drive version: 0.3.8.1
Commit Hash: 'aa79e324a22a2073b68c61783f4015426b7dbe67'
Go Version: go1.7.1
OS: linux/amd64
BuildTime: 2016-10-19 06:46:40.943196228 +0200 CEST

go version
go version go1.7.3 linux/amd64

git log -1
commit 9ce18db69196f53da028fab0ba9621dc33a23656
Author: Emmanuel Odeke <emm.odeke@gmail.com>
Date:   Sun Nov 6 22:20:00 2016 -0800
```
So:
- reported go version is inaccurate
- reported commit hash is also inaccurate

Also, the issue of this thread is still there.

**Comment by odeke-em** [comment]
Yeah if it was built from 9ce18db69, that's the latest.

As for the build, it seems like your build is from another binary or so, to build please try this
```shell
$ go get -u -v github.com/odeke-em/drive/drive-gen && drive-gen
```

In the worst case you can just into drive's repo and build by
```shell
$ cd $GOPATH/src/github.com/odeke-em/drive/cmd/drive && go build -o drive-fresh
$ ./drive-fresh list <whatever arguments>
```
or you can then checksum the freshly built source with your latest.
Also note that your $GOPATH/bin determines what drive binary you'll use in your path, the packaged version of Go at build time is what `drive version` reports.

Am using a Mac, and can't reproduce but that issue was definitely a problem before https://github.com/odeke-em/drive/pull/768.

**Comment by jean-christophe-manciot** [comment]
My go is not in $GOPATH/bin:
```
ll $GOPATH/bin/go*
-rwxr-x--- 1 actionmystique actionmystique 2460475 Jul 23 17:01 /home/actionmystique/Program-Files/Ubuntu/GO-Packages/bin/go-bindata*
-rwxr-x--- 1 actionmystique actionmystique 1997121 Jul 23 17:01 /home/actionmystique/Program-Files/Ubuntu/GO-Packages/bin/go-bindata-assetfs*
-rwxr-x--- 1 actionmystique actionmystique 7862620 Jul 31 08:37 /home/actionmystique/Program-Files/Ubuntu/GO-Packages/bin/godeps*
-rwxr-x--- 1 actionmystique actionmystique 6198096 Apr  1  2016 /home/actionmystique/Program-Files/Ubuntu/GO-Packages/bin/godex*
-rwxr-x--- 1 actionmystique actionmystique 5325304 Apr  1  2016 /home/actionmystique/Program-Files/Ubuntu/GO-Packages/bin/goimports*
-rwxr-x--- 1 actionmystique actionmystique 7506320 Apr  1  2016 /home/actionmystique/Program-Files/Ubuntu/GO-Packages/bin/gomvpkg*
-rwxr-x--- 1 actionmystique actionmystique 6817368 Apr  1  2016 /home/actionmystique/Program-Files/Ubuntu/GO-Packages/bin/gorename*
-rwxr-x--- 1 actionmystique actionmystique 6986984 Apr  1  2016 /home/actionmystique/Program-Files/Ubuntu/GO-Packages/bin/gotype*
-rwxr-x--- 1 actionmystique actionmystique 3090544 Jul 23 17:01 /home/actionmystique/Program-Files/Ubuntu/GO-Packages/bin/gox*
```
but rather in $GOROOT/bin (=$GOBIN) as pointed out by the official [go documentation](https://golang.org/doc/install/source#environment):
```
ll $GOROOT/bin/go*
-rwxr-xr-x 1 root root 9953852 Nov 11 13:27 /usr/lib/go/bin/go*
-rwxr-xr-x 1 root root 3036195 Nov 11 13:27 /usr/lib/go/bin/gofmt*

which go
/usr/lib/go/bin/go
```
Anyway, I have only one go binary on my system.

Regarding the issue of this thread, I compile the sources with ```drive-gen drive-google``` as mentioned by the documentation for Debian (I understood "debianish") distributions.
Trying without drive-google, the built is different. Now, ```drive list``` tries to list all folders, then all files per folder (although there's a glitch, one folder was missing 2 ---more---).
I understand the intent, but when the number of files per folder is more important than the size of a page, we have to manually ```| more```...

Also, without drive-google, the commit hash is now correct, but the go version remains incorrect:
```
drive version
drive version: 0.3.8.1
Commit Hash: '9ce18db69196f53da028fab0ba9621dc33a23656'
Go Version: go1.7.1
OS: linux/amd64
BuildTime: 2016-11-22 09:45:09.58465195 +0100 CET
```

**Comment by jean-christophe-manciot** [comment]
I close this issue since it has been solved by commits fresher than tag 0.3.8.1, but I reproduce latest post in related [thread](https://github.com/odeke-em/drive/issues/786#issuecomment-262207481).
